spec/features/mechanics/show_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'Mechanic Show Page', type: :feature do
+  before :each do
+    @park = AmusementPark.create!(name: 'Six Flags', admission_cost: 75)
+    @ride1 = @park.rides.create!(name: 'The Frog Hopper', thrill_rating: 5, open: true)
+    @ride2 = @park.rides.create!(name: 'Fahrenheit', thrill_rating: 8, open: true)
+    @new_ride = @park.rides.create!(name: 'New Ride', thrill_rating: 7, open: true)
+    @mechanic = Mechanic.create!(name: 'Kara Smith', years_experience: 11)
+    @mechanic.rides << @ride1
+    @mechanic.rides << @ride2
+  end
+
+  it 'shows the mechanic name, years of experience, and names of all rides they are working on' do
+    visit mechanic_path(@mechanic)
+
+    within('.mechanic-info') do
+      expect(page).to have_content(@mechanic.name)
+      expect(page).to have_content(@mechanic.years_experience)
```

Comment:
Although this assertion that checks for the years of experience is within a `within` block, I am still weary for the potential for this to cause a false positive. When looking for number values on a page, I would really like to see anchor/label text used to separate this number from any other random number on the page.

```ruby
expect(page).to have_content("Years of Experience: #{@mechanic.years_experience}")
```